buildkite/run_check_deploy_docker_build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This script checks that the image from the heroku-deploy Dockerfile
+# can be built successfully. This check is done in the buildkite-assets repo
+# to block merging/deploys to buildkite EC2 instances.
```

Comment:
> This check is done in the buildkite-assets repo to block merging/deploys to buildkite EC2 instances.

is a little confusing to me (we don't do this check _to_ block deploys, I think we do it to prevent broken deploys). How do you feel about a rewording along the lines of:
> This check is done in the buildkite-assets repo to prevent an unbuildable Dockerfile from being deployed to buildkite instances.

Reply:
Sounds good